velox/substrait/SubstraitToVeloxExpr.h

```diff
@@ -63,6 +64,14 @@ class SubstraitVeloxExprConverter {
       const RowTypePtr& inputType);
 
  private:
+  /// The entry of constructing ArrayVector from list literal
+  ArrayVectorPtr literalsToArrayVector(
```

Comment:
literalsToArrayVector and getArrayVectorFromListLiteral methods are very similar. Why do we need both of these? Can't we have just one?

Reply:
I'll give it a try.

Reply:
I tried it, it can be deleted.